decko-rails/lib/decko/rails.rb

```diff
@@ -0,0 +1,15 @@
+DECKO_RAILS_GEM_ROOT = File.expand_path('../../..', __FILE__)
+
+module Decko
+  module Rails
```

Comment:
I think (a) we will likely want to have a module based on the gem name for auto-require, and (b) this is the correct location:

http://guides.rubygems.org/name-your-gem/

`def root` may make more sense in the engine, but `def gem_root` belongs here, imo.